models/git_diff_test.go

```diff
@@ -139,3 +144,66 @@ func TestDiffLine_GetCommentSide(t *testing.T) {
 	assert.Equal(t, "previous", (&DiffLine{Comments: []*Comment{{Line: -3}}}).GetCommentSide())
 	assert.Equal(t, "proposed", (&DiffLine{Comments: []*Comment{{Line: 3}}}).GetCommentSide())
 }
+
+func benchParsePatch(b *testing.B, diffStr string) {
+	log.DelLogger("console")
+	log.DelLogger("file")
+	b.ResetTimer() //Disable logger for becnh
+	for i := 0; i < b.N; i++ {
+		ParsePatch(1000, 5000, 100, strings.NewReader(diffStr))
+	}
+}
+
+func BenchmarkParsePatchSimple(b *testing.B) {
```

Comment:
@sapk you wrote these tests IIRC, would you be able to help take a look into why they are failing?

Reply:
It seems that ParsePatch doesn't work as it return no change (0 line, 0 file). Looking into it.

Reply:
I look at it a little more but it doesn't seems obvious. I will need to run it step by step.

Reply:
I haven't had time to look at it.